### DISCUSSION TITLE
How do I install remark-highlight.js using Yarn?

### DISCUSSION THREAD

**Discussion by vinicius507** [remarkjs/remark#599] [Q&A] [answered]
The title is pretty self-explanatory. I'm using Yarn to manage my blog but when I try `yarn add remark-highlight.js` it says that the package is not found.

**Comment by ChristianMurphy** [comment, marked as answer]
`yarn add remark-highlight.js` is correct. https://www.npmjs.com/package/remark-highlight.js
Check you network connection, and check that the registry isn't down https://yarnpkg.com/getting-started/qa#queries-to-registryyarnpkgcom-return-a-404500-is-it-down
